packages/xstate-react/src/createHooks.ts

```diff
@@ -0,0 +1,106 @@
+import { createContext, useContext } from 'react';
+import { AnyActorRef, EmittedFrom } from 'xstate';
+import { useActor } from './useActor';
+import { useSelector } from './useSelector';
+
+/**
```

Comment:
This gets pretty lengthy and reiterates over what particular functions do - I wonder if it wouldn't be better to keep this succinct, only list what is being returned or smth and leave the rest for the descriptions of those functions that are being  returned